### ISSUE TITLE
Spell names on staves and books for some languages have gender forms and need a seperate suffix string to the spell string

### ISSUE THREAD

**Issue by xGnoSiSx** [diasurgical/devilutionX#4001]
Right now the code just adds the spell name string, but for Greek and some other languages, the words have different gender forms and also need a pronoun.

temporary fix I use is to have the neuterl noun and add quotes to the spell name:
   the "Town Portal"

It is wrong and not passable, as it looks like a tag, so long term we need stave spells to be proper suffixes with seperate strings to spellbook strings. There's no issue with scrolls, since they allready have seperate strings.

Town portal - is feminine
Golem - is neuter
Firebolt - is masculine
etc...

I had the same problem with books, and resolved to use 
Book: {Spell name}
or
Book With Title: {Spell Name}

The new suffixes would be used for books too, as they would include the correct pronoun and gendered form of the spell name.

**Comment by yuripourre** [comment]
Yes. This is one of the issues we have for most (maybe all) Latin languages. You are suggesting us to have a "duplicate spell name" that will only be applied to books, right?

I guess we could generalize it a bit more. Maybe we could have a "spell" text and an "of spell" text. e.g:
```
Spell: (_spell_) -> Spell: Healing
Scroll of (_of_spell_) -> Scroll of Healing
Book of (_of_spell_) -> Book of Healing
```
In English they could be the same but for other languages we could move the "of " to the translation `(_of_spell)` having something like:
```
Spell: (_spell_) -> Spell: Healing
Scroll (_of_spell_) -> Scroll of Healing
Book (_of_spell_) -> Book of Healing
```

This way it will also work for other languages.

**Comment by xGnoSiSx** [comment]
It's an additional string for books, scrolls and most importantly staves, applied as the same suffix.

Yes, for localizations that don't define it or need it, the code uses the spell_name.

Example for healing in greek:
Healing (Spell):  "Θεραπεία"
_spell_ = "Θεραπεία"
_spell_suffix_ = "της Θεραπείας"
Book of Healing (Book): "Βιβλίο της Θεραπείας"
Staff of Healing (Staff): "Ράβδος της Θεραπείας"
Scroll of Healing (Scroll): "Πάπυρος της Θεραπείας"

Then localization strings template 859 and 860 become obsolete, as staves. books, and scrolls can use the magical items templates as well :)
859:  {0} of {1}
860:  {0} {1} of {2}
